0.5/platform/web-animations.md

```diff
@@ -55,36 +53,32 @@ a `<div>` over 0.5 seconds. The animation alternates producing a pulsing effect.
       });
     </script>
 
+For more information on this `Animatable` interface, see the [specification](https://w3c.github.io/web-animations/#the-animatable-interface). It's supported natively in Chrome and Opera, and available with the [web-animations.min.js polyfill](https://github.com/web-animations/web-animations-js#web-animationsminjs) on other modern browsers.
+
 ## The animation model
 
-The Web Animations model is a description of an engine for animation content on the web. The engine is sufficiently powerful to support CSS Transitions, CSS Animations and SVG Animations.
+The Web Animations model is a description of an engine for animation content on the web. In browsers like Chrome, the engine is sufficiently powerful to support CSS animations.
 
 Web Animations also exposes a JS API to the model. This API defines a number of
 new interfaces that are exposed to JavaScript. We'll go through some of the more
-important ones here: Animations, AnimationEffects, TimingDictionaries, TimingGroups, and AnimationPlayers.
-
-An `Animation` object defines a single animation effect that applies to a single element target. For example:
+important ones here: `KeyframeEffect`, `AnimationEffectReadOnly`, `AnimationEffectTiming`, and `Animation`.
 
-    var animation = new Animation(targetElement,
-        [{left: '0px'}, {left: '100px'}], 2000);
+## Specifying animation effects
 
-Here, the target element's "left" CSS property is modified smoothly from `0px` to `100px` over 2 seconds.
+A `KeyframeEffect` object defines a single keyframe effect that applies to a single element target. For example:
 
-## Specifying animation effects
+    var effect = new KeyframeEffect(targetElement,
+        [{left: '0px'}, {left: '100px'}], 2000);
 
-An `AnimationEffect` object controls which CSS properties and SVG attributes are
-modified by an animation, and the values that those properties and attributes
-vary between. AnimationEffect objects also control whether the effect replaces
-or adds to the underlying value.
+Here, the target element's "left" CSS property is modified smoothly from `0px` to `100px` over 2 seconds. It's a subclass of `AnimationEffectReadOnly`, which provides timing configuration.
 
-There are three major kinds of effects: `KeyframeEffect`, `MotionPathEffect`, and `EffectCallback`.
+Note! Animating the "left" property is good for simple sites, but it will cause the browser to run layout on every frame. Regardless of your approach to animation, try to stick to animating "tranform" and "opacity" for the greatest performance.
```

Comment:
we should just show this as the example :) or update the example to use opacity or something else.